include/deal.II/base/parameter_handler.h

```diff
@@ -1558,7 +1558,35 @@ class ParameterHandler : public Subscriptor
     ShortText = 193
   };
 
+  /**
+   * Flags to control input file scanning behavior.
+   */
+  enum ReadInputFlags
+  {
+    /**
+     * A flag for no flag.
+     */
+    ReadInputFlag_Null = 0,
```

Comment:
We typically put the enum into a ```struct``` to group them together and then don't preface each entry.

A better name here would be "default"? The documentation should also state that this sets the default values.